File: bench/sort.js
const arr = Array.from({length: 10000}, () => Math.random() * 10000);

const start = Date.now();
const sorted = [...arr].sort((a, b) => a - b);
const elapsed = Date.now() - start;

console.log(`Sort 10k random numbers in ${elapsed}ms`);
console.log(`First 5: ${JSON.stringify(sorted.slice(0, 5))}`);

File: bench/sort.rb
arr = Array.new(10000) { rand * 10000 }

start = (Time.now.to_f * 1000).to_i
sorted = arr.sort
elapsed = (Time.now.to_f * 1000).to_i - start

puts "Sort 10k random numbers in #{elapsed}ms"
puts "First 5: #{sorted[0..4].inspect}"
